core/os/android/adb/device.go

```diff
@@ -217,6 +217,11 @@ func newDevice(ctx context.Context, serial string, status bind.Status) (*binding
 		d.To.Configuration.PerfettoCapability = perfettoCapability
 	}
 
+	// Query device ANGLE support
+	if anglePackage, err := d.QueryAnglePackageName(ctx); err == nil {
+		d.To.Configuration.AnglePackage = anglePackage
```

Comment:
This is meant to set a variable based on the queried angle package name.
However, what I don't understand is what is the angle_package string that I put into device.proto used for? Do I need to fill that in here? If so, how and why?

Reply:
Yes. The AnglePackage set here can then be retrieved in the UI code so that you know which package is enabled for the selected device. This can then allow you to either show Angle trace type or not, based on the selected device's configuration.